benchmarks/torchbench.py

```diff
@@ -7,18 +7,23 @@
 import sys
 import warnings
 from os.path import abspath
+from os.path import dirname
 from os.path import exists
+from unittest.mock import patch
 
 import torch
-from common import BenchmarkRunner
-from common import main
 
 from torchdynamo.testing import collect_results
 from torchdynamo.testing import reduce_to_scalar_loss
 from torchdynamo.utils import clone_inputs
 
+with patch.object(sys, "path", [abspath(dirname(__file__))] + sys.path):
```

Comment:
For fbcode, can we just fix the buck targets for this to get the right deps in place, instead of patching the sys path?

Reply:
I don't know buck well enough to know how to do this.  You want to give it a try?

changing this to `.common` works in fbcode, by not in oss.